nemo/collections/asr/models/wav2vec/modules/multihead_attention.py

```diff
@@ -0,0 +1,396 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
NVIDIA licence. Would you please add the link to the main file this code is adopted from?